tests/VirtoCommerce.ExperienceApiModule.Tests/Index/IndexSearchRequestBuilderTests.cs

```diff
@@ -347,6 +347,61 @@ public void BuildRequestFromDefault_ReturnDefaultRequest()
             actual.Should().BeEquivalentTo(expected);
         }
 
+
+        #region Sorting
+
+        [Theory]
```

Comment:
Need to revise test logic.
We need to ensure that in case of no `AddSorting` method or in case of null sorting we will get in the request is Score desc

Reply:
Yes, I think we test this https://github.com/VirtoCommerce/vc-module-experience-api/blob/e8f292db76687b4dfa091e6a0863be07f0c5df4a/tests/VirtoCommerce.ExperienceApiModule.Tests/Helpers/MoqHelper.cs#L40

Reply:
In all cases here we are testing `AddSorting`. There are no tests without it.

Reply:
Refactored